php/hudson.php/src/org/netbeans/modules/hudson/php/options/HudsonOptions.java

```diff
@@ -36,7 +36,7 @@ public final class HudsonOptions {
 
     // Do not change arbitrary - consult with layer's folder OptionsExport
     // Path to Preferences node for storing these preferences
-    private static final String PREFERENCES_PATH = "hudson"; // NOI18N
+    private static final String PREFERENCES_PATH = "jenkins"; // NOI18N
```

Comment:
this is something we will have to test.

config migration and config import/export should still work after this change.

IMO: migration of hudson -> jenkins is probably not worth to implement, what I mean by that is to check if the first-launch settings import from NB 21 to NB 22 still works if this path changes.

It might be better to not change this property and move that task for later - I am not sure. (thats why I advocated to rename everything - since this does make things easier)

Reply:
Agreed. This does not worth the change. Changing this would mean that users invested in customizing their settings would lose those, unless handled in the updater logic.